robot/Cumulus/tests/browser/zpayment_allocations/enable_pmt_alloc_and_test.robot

```diff
@@ -12,49 +15,72 @@ Suite Teardown  Run keywords
 ...             Disable Payment Allocations
 ...             Delete Records and Close Browser
 
+*** Variables ***
+&{contact1_fields}         Email=test@example.com
+&{contact2_fields}         Email=test@example.com
+&{opportunity1_fields}     Type=Donation   Name=Opp Allocation Sync test Donation   Amount=100  StageName=Prospecting    npe01__Do_Not_Automatically_Create_Payment__c=false
+&{opportunity2_fields}     Type=Donation   Name=Payment Allocation Sync test Donation   Amount=100  StageName=Prospecting    npe01__Do_Not_Automatically_Create_Payment__c=false
+
 *** Test Cases ***
 
 Create Payment Allocations and Verify Opportunity Allocations Sync
-    [tags]    unstable
-    Go To Record Home  &{opportunity}[Id]
-    Select Tab    Related
-    Load Related List    GAU Allocations
-    Click Link With Text    &{payment}[Name]    
+    [Documentation]                        Once Allocations are Enabled and a Default GAU is added, create an opportunity via API.
+    ...                                    Open Opportunity and open payments from the related tab. Verify that total payment is assigned to Default GAU
+    ...                                    Add a new Allocation to another GAU and verify that the Allocation is split between default and new GAUs.
+    ...                                    Go to Opportunity and verify that same allocation is reflected under GAU allocations related list. 
+    [tags]                                 W-039821            feature:Payment Allocations
+    Go To Page                             Detail
+    ...                                    Opportunity
+    ...                                    object_id=${data}[contact1_opportunity][Id]
+    Select Tab                             Related
+    Load Related List                      GAU Allocations
+    Click Link With Text                   &{payment1}[Name]    
     Select Window
```

Comment:
You are not passing value of the window? is this intentional?

Reply:
yes, it selects the active window by default